<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screenShotClass</class>
 <widget class="QWidget" name="screenShotClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>screenShotClass</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>180</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileDir">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getDirBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bmp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>png</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
